### PR TITLE
Create PRS12936WaldaSamaet.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New record

### CODE REVIEW

File: new/PRS12936WaldaSamaet.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        </profileDesc>
+        <revisionDesc>
+            <change who="DN" when="2020-03-30">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">WaldaSamaet</persName>
```
Comment (eu-genia) [inline new/PRS12936WaldaSamaet.xml]: fidel name

File: new/PRS12936WaldaSamaet.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
```
Comment (eu-genia) [inline new/PRS12936WaldaSamaet.xml]: transcribed name

File: new/PRS12936WaldaSamaet.xml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic"/>
+                    <nationality type="Ethiopia"/>
+                    <birth/>
+                    <death/>
+                    <floruit/>
+                </person>
+                <listRelation>
+                    <relation name="saws:hasOwned" active="PRS12936WaldaSamaet" passive=""/>
```
Comment (eu-genia) [inline new/PRS12936WaldaSamaet.xml]: passive relation?

File: new/PRS12936WaldaSamaet.xml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">WaldaSamaet</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
+                    <faith type="EOTC"/>
+                    <occupation type="ecclesiastic"/>
+                    <nationality type="Ethiopia"/>
+                    <birth/>
+                    <death/>
+                    <floruit/>
```
Comment (eu-genia) [inline new/PRS12936WaldaSamaet.xml]: add floruit dates for orientation

File: new/PRS12916MazraetaKrestos.xml [hunk 5]
```diff
@@ -45,18 +45,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <body>
             <listPerson>
                 <person sex="1">
-                    <persName xml:lang="gez" xml:id="n1">MazraetaKrestos</persName>
+                    <persName xml:lang="gez" xml:id="n1">መዝራዕተ᎓ ክርስቶስ</persName>
                     <persName xml:lang="gez" type="normalized" corresp="#n1"/>
+                    <persName type="alt"><roleName type="title">ʾabbā</roleName>Mazrāʿǝta Krǝstos</persName>
```
Comment (eu-genia) [inline new/PRS12916MazraetaKrestos.xml]: this is not type="alt" this should go into the type="normalized" tag

File: new/PRS12916MazraetaKrestos.xml [hunk 6]
```diff
@@ -45,18 +45,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
                     <faith type="EOTC"/>
                     <occupation type="ecclesiastic"/>
                     <birth/>
                     <death/>
                     <floruit/>
```
Comment (eu-genia) [inline new/PRS12916MazraetaKrestos.xml]: add dates for floruit for orientation

Comment (DenisNosnitsin1970) [reply new/PRS12916MazraetaKrestos.xml]: Ok

File: new/PRS12936WaldaSamaet.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/PRS12936WaldaSamaet.xml]: Done

File: new/PRS12936WaldaSamaet.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/PRS12936WaldaSamaet.xml]: Ok

File: new/PRS12936WaldaSamaet.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/PRS12936WaldaSamaet.xml]: Ok

File: new/PRS12936WaldaSamaet.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply new/PRS12936WaldaSamaet.xml]: Ok

File: new/PRS12916MazraetaKrestos.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply new/PRS12916MazraetaKrestos.xml]: I followed the "Guidelines".

File: new/PRS12936WaldaSamaet.xml [hunk 7]
```diff
@@ -45,19 +45,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         <body>
             <listPerson>
                 <person sex="1">
-                    <persName xml:lang="gez" xml:id="n1">WaldaSamaet</persName>
-                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
+                    <persName xml:lang="gez" xml:id="n1">ወልደ᎓ ሰማዕት</persName>
```
Comment (eu-genia) [inline new/PRS12936WaldaSamaet.xml]: word division sign at the end

File: new/PRS12936WaldaSamaet.xml [hunk 8]
```diff
@@ -45,19 +45,19 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Walda Samāʿǝt</persName>
                     <faith type="EOTC"/>
                     <occupation type="ecclesiastic"/>
                     <nationality type="Ethiopia"/>
                     <birth/>
                     <death/>
-                    <floruit/>
+                    <floruit>Late 18th to 1840</floruit>
```
Comment (eu-genia) [inline new/PRS12936WaldaSamaet.xml]: notBefore="1775" notAfter="1840"
Late 18th century

File: new/PRS12916MazraetaKrestos.xml (hunk 5 above)
Comment (eu-genia) [reply new/PRS12916MazraetaKrestos.xml]: roleName can be an extra persName element but you must have the normalized version of the fidel name for proper encoding and visualization

File: new/PRS12936WaldaSamaet.xml (hunk 7 above)
Comment (DenisNosnitsin1970) [reply new/PRS12936WaldaSamaet.xml]: Done

File: new/PRS12916MazraetaKrestos.xml (hunk 6 above)
Comment (DenisNosnitsin1970) [reply new/PRS12916MazraetaKrestos.xml]: Done

File: new/PRS12936WaldaSamaet.xml (hunk 8 above)
Comment (DenisNosnitsin1970) [reply new/PRS12936WaldaSamaet.xml]: Done

File: new/PRS12916MazraetaKrestos.xml (hunk 5 above)
Comment (DenisNosnitsin1970) [reply new/PRS12916MazraetaKrestos.xml]: Ok

**Comment by thea-m** [comment]
I did not read these files because they seem to be already thouroughly reviewed, but I have a general remark to @eu-genia (and the rest of us). While I completely see the usefulness of adding floruit ranges to person records, I would not insist on adding these as a general rule, since this would, in my impression, bring us closer to the undesired "here is a form with many fields which we fill in" approach. It is good to be reminded that they might be usefully added, but could still be decided case by case by the encoder.

**Comment by thea-m** [comment]
And note that this PR has a very simple conflict that need resolving and would be ideal for practicing this crucial skill :) If anyone is unsure how to do this but would like to practice it, you can contact me and we can do it together.

**Comment by DenisNosnitsin1970** [comment]
Some of the persons should be dissimilated, at least as rough as it is through <floruit>.   I did it because I know another Mazra'eta Krestos, who lived in the time befor the one above. Any cognitive help to distinguish between anything/anyone will be later greatly appreciated by the users.

**Comment by eu-genia** [comment]
It is not that it is a "must fill" field, but this is the only way to at least roughly disambiguate the many homonymous persons (also scribes - I have many scribes in E-S by the same name but in different periods)

**Pull request merged**